test_util/helpers.py

```diff
@@ -31,6 +30,35 @@ def authenticate(self, cluster):
         logging.info('Authentication successful')
 
 
+class ApiClient:
+    def __init__(self, cluster, user, api_base, default_headers={}):
```

Comment:
python doesn't like passing dictionaries as default arguments, should make the default `None` and do a `if default_headers is None: default_headers = dict()`